vaadin-crud-flow-integration-tests/src/test/java/com/vaadin/flow/component/crud/test/ClickRowToEditIT.java

```diff
@@ -0,0 +1,84 @@
+package com.vaadin.flow.component.crud.test;
+
+import com.vaadin.flow.component.button.testbench.ButtonElement;
+import com.vaadin.flow.component.confirmdialog.testbench.ConfirmDialogElement;
+import com.vaadin.flow.component.crud.examples.ClickRowToEditView;
+import com.vaadin.flow.component.crud.testbench.CrudElement;
+import com.vaadin.flow.component.grid.testbench.GridElement;
+import com.vaadin.flow.component.textfield.testbench.TextFieldElement;
+import com.vaadin.testbench.TestBenchElement;
+import com.vaadin.testbench.annotations.RunLocally;
+import com.vaadin.testbench.parallel.Browser;
+import com.vaadin.testbench.parallel.BrowserUtil;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+@RunLocally(Browser.CHROME)
+public class ClickRowToEditIT extends AbstractParallelTest {
+
+    @Before
+    public void init() {
+        getDriver().get(getBaseURL() + "/clickrowtoedit");
+    }
+
+    @Override public void setup() throws Exception {
+        super.setup();
+        System.setProperty("webdriver.chrome.driver", "/User/diegocardoso/Tools/chromedriver");
```

Comment:
Should be updated. I guess it's a leftover :)

Reply:
It was. Removed. :)